pipelines/controller/src/main/java/com/google/fhir/analytics/HiveTableManager.java

```diff
@@ -128,20 +129,21 @@ private synchronized void createViews(Connection connection, String resource) {
     if (viewsDir.isEmpty()) {
       return;
     }
-    List<Path> viewPaths = null;
-    try {
+    List<Path> viewPaths;
+    try (Stream<Path> stream = Files.list(Paths.get(viewsDir))) {
```

Comment:
Hmm, is there a `close` that needs to be done on this? And did ErrorProne gave you a warning for this?